README.md

```diff
@@ -19,10 +19,12 @@ A list of missing exercise can be found here: http://exercism.io/languages/pytho
 
 ### Conventions
 
-- We use minimalistic stub files for all exercises (#272).
+- We use minimalistic stub files for all exercises ([#272](https://github.com/exercism/python/issues/272)).
```

Comment:
GitHub implicitly links #272 to issue 272 in same repository; is it really necessary to change to explicit links?

Reply:
On the GitHub repo page, these don't appear to be implicitly linked - I probably should have made that clear in the PR.

Reply:
You are right... in fact, they are not implicitly linked even if you click on the README.md file itself. not sure why issue number syntax is implicitly linked everywhere on GitHub except for when viewing actual markdown files. Might be a GitHub bug... good catch.